javascript/javascript2/week1/spiritAnimalNameGenerator.js

```diff
@@ -0,0 +1,54 @@
+
+const spiritAnimalNameList = [
+    'Sensible Fuchsia Mocking Bird',
+    'Steady Plumboa',
+    'Purple Cottontail',
+    'Contemplative Tea Beetle',
+    'Big eyes rhinos',
+    'the fullmoon wolf',
+    'Cute blue cat',
+    'the sharp shark',
+    'long tailed monkey',
+    'so cute white dove'
+];
+
+function spiritAnimalNameGenerator(name) {
+    return `${spiritAnimalNameList[parseInt(Math.random() * 10)]}`;
+}
+
+function spiritAnimalOutput() {
+    const getName = document.querySelector('#name');
+    const message = document.querySelector('#spiritName');
+    if (getName.value !== '') {
+        message.innerHTML = `${getName.value} - ${spiritAnimalNameGenerator(getName)}`;
+    } else {
+        message.innerHTML = 'Provide a name:';
```

Comment:
Good thinking.

Reply:
can i write just:
 if(getName.value){
               message.innerHTML = `${getName.value} - ${spiritAnimalNameGenerator(getName)}`;
            }
instead of line 22